lib/url.c

```diff
@@ -6238,9 +6238,12 @@ static CURLcode resolve_server(struct Curl_easy *data,
       if(rc == CURLRESOLV_PENDING)
         *async = TRUE;
 
-      else if(rc == CURLRESOLV_TIMEDOUT)
+      else if(rc == CURLRESOLV_TIMEDOUT) {
+        failf(data, "Failed to resolve host '%s' with timeout after %ld ms",
+              connhost->dispname,
+              Curl_tvdiff(now, data->progress.t_startsingle));
```

Comment:
How spammy is this log?  Is it OK?

Reply:
`failf` will just log it to a buffer - it's the calling code's responsibility whether it logs it out. 

Less pedantically, all of the other paths which set a failure timeout DO log, this one doesn't, which is irritating. We (Sprout) currently drop a warning log on every CURL failure, this just means it logs something useful.